### DISCUSSION TITLE
essay/2024/07/25/kakao-fire-guys

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#224] [Comments]
# essay/2024/07/25/kakao-fire-guys

오랜만에 옛 동료들과 점심을 먹고 수다를 떨다가 들어왔습니다. 카카오 아지트 방문.

https://jeho.page/essay/2024/07/25/kakao-fire-guys.html

**Comment by Hyunhyun12** [comment]
헐 소프트모히칸 ㅋㅋㅋㅋㅋ 그때 그시절 유행하던 헤어스타일이었죠! 멋쟁이 동료분께서는 지금도 유행하는 헤어스타일을 하고 계시네요! ㅎㅎㅎ

**Reply by BenjaminKim** [reply]
하하, 소프트모히칸이란 이름이 있나요? 말씀을 듣고 자세히 보니 헤어 스타일에 신경을 많이 썼네요. 그동안 전혀 몰랐습니다. 😁
